woocommerce/woocommerce-gutenberg-products-block
# Remove Shipping calculator settings and link to the WooCommerce admin settings

## What

Removes block level settings that enable/disable the shipping calculator and whether costs are displayed in favour of a link to the main settings page. 

This PR also clears up the verbiage used in other areas to keep text consistent. This includes:

- Inconsistent use of "the woocommerce settings" vs "store settings"
- "options" vs "settings". Since "options" conflicted with shipping options (rates) I avoided the term completely

See screenshots for the new panels.

This does introduce the side effect of "losing" the block level setting, but we cannot really sync this back up because it would affect other blocks and the classic checkout. Should only be a minor inconvenience.

Fixes #11010

## Why

See https://github.com/woocommerce/woocommerce-blocks/issues/11010

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to a page containing the Cart Block
2. Highlight the shipping costs row.
3. You should see a panel called "Shipping Calculations" with a link to the shipping options page
4. Go to a page containing the Checkout block
5. Highlight the shipping methods block (the shipping/local pickup toggle).
6. You should only see the shipping methods panel here which links to zones/local pickup
7. Highlight the shipping options block.
8. You should see a panel called "Shipping Calculations" with a link to the shipping options page


## Screenshots or screencast

Cart:
![Screenshot 2023-10-09 at 14 16 29](https://github.com/woocommerce/woocommerce-blocks/assets/90977/723ef318-e44b-475e-b61f-4a650b238b33)

Checkout Shipping Methods:
![Screenshot 2023-10-09 at 14 18 30](https://github.com/woocommerce/woocommerce-blocks/assets/90977/d311d572-dfad-4625-aaee-3e54ed4cb73f)

Checkout Shipping Options:
![Screenshot 2023-10-09 at 14 18 44](https://github.com/woocommerce/woocommerce-blocks/assets/90977/b6c750b4-7041-4b0a-ba70-5105f805073d)

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Removed block level shipping calculator and hide costs option in favour of the global setting
